prereise/gather/griddata/hifld/const.py

```diff
@@ -118,3 +120,129 @@
     "345": 345,
     "500": 500,
 }
+
+interconnect2state = {
+    "east": {
+        "AL",
+        "AR",
+        "CT",
+        "DE",
+        "FL",
+        "GA",
+        "IA",
+        "IL",
+        "IN",
+        "KS",
+        "KY",
+        "LA",
+        "MA",
+        "MD",
+        "ME",
+        "MI",
+        "MN",
+        "MO",
+        "MS",
+        "NC",
+        "ND",
+        "NE",
+        "NH",
+        "NJ",
+        "NY",
+        "OH",
+        "OK",
+        "PA",
+        "RI",
+        "SC",
+        "TN",
+        "VA",
+        "VT",
+        "WI",
+        "WV",
+    },
+    "ignore": {"AK", "HI"},
+    "split": {"MT", "SD", "TX"},
+    "west": {"AZ", "CA", "CO", "ID", "NM", "NV", "OR", "UT", "WA", "WY"},
+}
+
+state_county_splits = {
+    "MT": {
+        "default": "west",
+        "east": {
+            "CARTER",
+            "CUSTER",
+            "DANIELS",
+            "DAWSON",
+            "FALLON",
+            "GARFIELD",
+            "MCCONE",
+            "PHILLIPS",
+            "POWDER RIVER",
+            "PRAIRIE",
+            "RICHLAND",
+            "ROOSEVELT",
+            "ROSEBUD",
+            "SHERIDAN",
+            "VALLEY",
+            "WIBAUX",
+        },
+    },
+    "NM": {"default": "west", "east": {"CURRY", "LEA", "QUAY", "ROOSEVELT", "UNION"}},
+    "SD": {"default": "east", "west": {"BUTTE", "FALL RIVER", "LAWRENCE"}},
+    "TX": {
+        "default": "texas",
+        "east": {
+            "BAILEY",
+            "BOWIE",
+            "CAMP",
+            "CASS",
+            "COCHRAN",
+            "DALLAM",
+            "DONLEY",
+            "GAINES",
+            "GREGG",
+            "HALE",
+            "HANSFORD",
+            "HARDIN",
+            "HARRISON",
+            "HARTLEY",
+            "HEMPHILL",
+            "HOCKLEY",
+            "HUTCHINSON",
+            "JASPER",
+            "JEFFERSON",
+            "LAMB",
+            "LIBERTY",
+            "LIPSCOMB",
+            "LUBBOCK",
+            "LYNN",
+            "MARION",
+            "MOORE",
+            "MORRIS",
+            "NEWTON",
+            "OCHLTREE",
+            "ORANGE",
+            "PANOLA",
+            "PARMER",
+            "POLK",
+            "RANDALL",
+            "SABINE",
+            "SAN AUGUSTINE",
+            "SAN JACINTO",
+            "SHELBY",
+            "SHERMAN",
+            "TERRY",
+            "TRINITY",
+            "TYLER",
+            "UPSHUR",
+            "WALKER",
+            "YOAKUM",
+        },
+        "west": {"EL PASO", "HUDSPETH"},
+    },
+}
+
+hifld_storage_types = {
+    "BATTERIES",
+    "NATURAL GAS WITH COMPRESSED AIR STORAGE",
+    "FLYWHEELS",
+}
```

Comment:
I am wondering if at some point we don't want to move these constants in:
* `powersimdata/network/hifld/constants/plants.py`
* powersimdata/network/hifld/constants/zones.py
once they exist

Reply:
It could make sense, `const.py` is getting a bit long.